src/types/Health.ts

```diff
@@ -121,19 +123,33 @@ const ascendingThresholdCheck = (value: number, thresholds: Thresholds): Thresho
   return { value: value, status: HEALTHY };
 };
 
-export const getRequestErrorsRatio = (rh: RequestHealth): ThresholdStatus => {
-  if (rh.errorRatio < 0) {
+export const getRequestErrorsRatio = (rh: RequestHealth, propName?: string): ThresholdStatus => {
```

Comment:
propName could be `keyof RequestHealth` maybe?

Reply:
+1 I'll change